rasa/shared/nlu/training_data/schemas/responses.yml

```diff
@@ -11,6 +11,7 @@ schema;responses:
       - type: "map"
         required: True
         allowempty: False
+        matching: any
```

Comment:
Hey @bastbnl thank you for submitting a PR for this issue! I tried this change locally but still the error message is the same as before and does not provide further/more detailed information about why the `domain.yml` file is not valid. Did it work for you? I use a non valid `domain.yml` that just has the same invalid key that you used in the description of the issue [here](https://github.com/RasaHQ/rasa/issues/8625#issue-877880747).